pytest_playwright/pytest_playwright.py

```diff
@@ -134,7 +133,7 @@ def _handle_page_goto(
     page: Page, args: List[Any], kwargs: Dict[str, Any], base_url: str
 ) -> None:
     url = args.pop()
-    if not (url.startswith("http://") or url.startswith("https://")):
+    if not url.startswith(("http://", "https://")):
```

Comment:
The previous code in my mind was more readable. Also most resources in the internet never mention that it supports a tuple, so I would keep the approach which we had before.

Reply:
Ok!